packages/mobile/src/app/saga.ts

```diff
@@ -208,21 +180,24 @@ function* watchAppState() {
 
 function* handleSetAppState(action: SetAppState) {
   const appLocked = yield select(getAppLocked)
-  const cachedPin = getCachedPincode()
+  // When requesting Android Permissions app state
+  // would background, so we do not want to lock
+  // the in this case
```

Comment:
why not in this case?

Reply:
Cause, when user would get a request permission notification we do not want to lock the app (user has not left it or backgrounded really). I've updated the comment, so it's more specific on reasoning.